tasks/unit-tests/testdata/gitlab_mq_context_template.yml

```diff
@@ -0,0 +1,10 @@
+---
+variables:
+  BUCKET_BRANCH: 'dev'
+  CI_COMMIT_BRANCH: 'mq-working-branch-main'
+  CI_PIPELINE_SOURCE: 'pipeline'
+  DEPLOY_AGENT: 'false'
+  RUN_ALL_BUILDS: 'false'
+  RUN_E2E_TESTS: 'auto'
+  RUN_KMT_TESTS: 'off'
+  RUN_UNIT_TESTS: 'off'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Can you add a line feed ?